_pages/en_US/sd-card-setup.md

```diff
@@ -131,7 +131,7 @@ mmcblk0     179:0    0   3,8G  0 disk
 
 ___
 
-If the test shows the result `Data LOST: 0.00 Byte (0 sectors)` your SD card is good and you can delete all `.h2w` files on your SD card
+If the test shows the result `Data LOST: 0.00 Byte (0 sectors)` your SD card is good and you can delete all `.h2w` files on your SD card.
```

Comment:
I guess the use of the word good is ambiguous. I would change it honestly.

Reply:
Maybe using "healthy" is a good idea.